app/views/shared/_maintenance_window_alert.html.erb

```diff
@@ -0,0 +1,26 @@
+<%
+  maintenance_window = MaintenanceWindow.new(
+    start: Figaro.env.acuant_maintenance_window_start,
+    finish: Figaro.env.acuant_maintenance_window_finish,
```

Comment:
In the future, we can make these start/end variables we pass in, but I figured for the time being, we have just the one maintenance window, so I hardcoded it here